Update the official cnc/TD map pool

gdavegdave opened the pull request:
This is a suggested change to the CnC/TD official map pool, arrived at in consultation with many of the regular TD players. This pull request is in response to the request made by pchote here: http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=20031, which has discussion and a poll on the issue. I had raised an issue (https://github.com/OpenRA/OpenRA/issues/13016) but thought it would be more helpful to make the changes myself.

Below is a list of maps I suggest to add/modify/leave unchanged/remove, comparing with the maps in the playtest-20170304. In the case of maps to be added or modified, I have used 'my' filenames. In the case of maps to be removed, I have used the playtest-20170304 filenames. I have given brief reasons for each modification.

I changed the 'Categories' from '%' to 'Conquest' for the affected maps (all by Norman). I cleaned up the titles of some maps, but I left the map filenames and the map authors intact. As such, the UIDs for ~19 maps will now differ from the list below.


**Add**

1. 169.oramap da8e7c4d4c4ca62812af156769ee9544f403bdd0 http://resource.openra.net/maps/16886/
2. arcade.oramap aeeb0337abf307755a862ee3e321e804e623fdc4 http://resource.openra.net/maps/16895/
3. Blood_and_Sand.oramap b4a8ea0701db2874f3a7a5a50631617f0ecf99b2 http://resource.openra.net/maps/12728/
4. CarterRidge-1.oramap 7973ceac3053a3568a5746e1cca5d251d07d3059 http://resource.openra.net/maps/17571/
5. chillout.oramap 8692c62757df9d5bf3465937258a330cdd0e2466 http://resource.openra.net/maps/11901/
6. CollateralChaos5.oramap 4a4287267b15ede75b4387515c9f26950482c6b6 http://resource.openra.net/maps/19656/
7. Contra.oramap 2f43dd132333d620a58bf9f933d4f63e15c62beb http://resource.openra.net/maps/14000/
8. Crossing_The_Divide.oramap f46588829b7dca039cbb51b7c351fbcd128ba746 http://resource.openra.net/maps/17425/
9. dead_in_motion_2.oramap 11c706bf5f31d95d60182fffc27f531c139528c5 http://resource.openra.net/maps/20094/
10. desert_rats_td_v2.oramap eb4dddeb0927b1f1fa1dde53ebb1aa4c5caf7ddd http://resource.openra.net/maps/16879/
11. deterring_democracy.oramap 90391ffab4b4701f90b29bf4058276beb6576b5f http://resource.openra.net/maps/13237/
12. deterring_democracy_plus.oramap fb10704c557f17787a97c16d63c9143a1a387f14 http://resource.openra.net/maps/12608/
13. dualism.oramap 51853b5c481eae2a487b2909ee35aaa3d86f3d18 http://resource.openra.net/maps/11915/
14. Frigid_Pass.oramap 96fee4a172f9c44a3034bd79f95424c1773e9c15 http://resource.openra.net/maps/17910/
15. Frostways-6.oramap 7dd88799c41e61e5ff0a670203ae9b7db26e2f96 http://resource.openra.net/maps/20054/
16. hegemony_or_survival_8p.oramap cea558edabfa4cb4cdc130d9d1dc3c47727420c7 http://resource.openra.net/maps/11981/
17. IN-3PMap-7.oramap 2e35002f01bf5dad90173eae1d96f5268944336d http://resource.openra.net/maps/20032/
18. IN-Desert_Lakes_Unmodded_-7.oramap 1fc3353577846488c7e4efe0171fdacf85c68e0d http://resource.openra.net/maps/20024/
19. IN-FortressesIII-2.oramap df74b7a09a1933fb5fa8ceb6e94660ed100c4624 http://resource.openra.net/maps/20031/
20. jconflict.oramap 5d5f51a0cee29cdd4c104718994cd0170b7d3382 http://resource.openra.net/maps/13448/
21. letters_from_lexington.oramap 291e3707fca0c98bbbd6acccb180a23882717a49 http://resource.openra.net/maps/14560/
22. model150.oramap 1ab5ad77144d6342968030e9de2bce0e9a3a8871 http://resource.openra.net/maps/16943/
23. model200.oramap fc1ef853b38322644632606aeb117f5334754931 http://resource.openra.net/maps/16676/
24. mtnukebait.oramap 92b6d6a9572d4295e8ff2e82425c13594a58ebdd http://resource.openra.net/maps/20102/
25. planb.oramap cbef0c507caa1e39bcc2584624577b82f8f4c4b8 http://resource.openra.net/maps/16952/
26. reasonabledoubt.oramap b6cefe97e71a9015876c395177516e2d237e1510 http://resource.openra.net/maps/12615/
27. Scorched_Earth.oramap c9bdfd8f71a96472aa7b06d124ad3cdcd9b1b6d8 http://resource.openra.net/maps/18504/
28. Tactical_Advance.oramap e75c392ff23c327c25b61224b1524a649d893618 http://resource.openra.net/maps/17483/
29. ThawedFront-5.oramap f561aaea8ce10996d118aa3e4854a461f2e628d3 http://resource.openra.net/maps/18035/
30. the-hot-box-v2.oramap a44821288d43fd5ac73478eab3d71998f48d5988 http://resource.openra.net/maps/20083/
31. TibCanopy-9.oramap dab8fefee5db7e71c65083f77aa198f0673ab621 http://resource.openra.net/maps/18270/
32. Tiberian_Falls.oramap ee94bc34598b56237c7a42ec2a6d3be2d5ea002a http://resource.openra.net/maps/20034/
33. Trial_of_Possession.oramap a8630d3836b63e05c8f981027f49006375725f59 http://resource.openra.net/maps/17198/
34. Twin_Lakes.oramap d5d6262e6676b7a4168a929fd6bebc34226ea40e http://resource.openra.net/maps/17426/
35. two-ponds.oramap d5b1bfa40ebfc093ac29c4d7eced8e1dcbcee573 http://resource.openra.net/maps/13371/
36. understanding_power_v3.4_celeryfix.oramap e2c98e5eea32f4f206b5661c75e9c6a82c902339 http://resource.openra.net/maps/11824/
37. Warchild.oramap 95f2e7fc4e36e60be4c0aa64b63cb45bfb546f53 http://resource.openra.net/maps/17178/
38. Winters_End.oramap 901f0501bdf5f6651e0a8c7d94081dd66c2d81e5 http://resource.openra.net/maps/20055/
39. X896.oramap 084b4bddd86b79671048f3fd289e3c68c1ce4886 http://resource.openra.net/maps/13413/
40. year_501_rev.oramap 2e982bc1cfcf92e9c5e2f82319da74f2bb99d1ed http://resource.openra.net/maps/12395/
41. _lostsouls.oramap 294002f10fe55e802740bf3d4bd81488374fd968 http://resource.openra.net/maps/20104/


**Modify**

1. aggressivetendencies.oramap 6c3893a3c820ca680a038f6f33e5b1ac1466772d http://resource.openra.net/maps/20092/
...Revert to release-20161019 version, so as to revert author name from 'Pulse' to 'pulse'
2. alpha-juno.oramap 5978601ab16c81342718740a0b980fca59e965e6 http://resource.openra.net/maps/18591/
...Update to the preferred version
3. AxisAdvance5.oramap 20ff1bc2ba8147a505a2f8f6e19c8c78b1e60688 http://resource.openra.net/maps/19823/
...Update to latest, preferred version (oil in middle instead of blue trees)
4. backtobasics.oramap bc3b69c25bf57ea0738bd774366f7634a939305f http://resource.openra.net/maps/13423/
...Revert author name from 'Pulse' to 'pulse'
5. botany.oramap dbaa2606988ff9e3cbefd9a8572c946d7919538b http://resource.openra.net/maps/11906/
...Change to the preferred version
6. controlandchaos.oramap 449308b7191bce0e74f4adca69f12453abd80e6e http://resource.openra.net/maps/13265/
...Revert author name from 'Pulse' to 'pulse'
7. desert_storm_3.oramap 91695bb8ee84423d65d41a3b9a447dd4f8cbf0a9 http://resource.openra.net/maps/16907/
...Revert author name from 'General Motors' to 'general motors'
8. eastwest3.oramap d07e7ade8051ce0282ea254459189ad02df3f0f6 http://resource.openra.net/maps/12400/
...Revert map filename
9. escalations.oramap 3468a2fc1e4387cfd412f0b54ced46c0de76c068 http://resource.openra.net/maps/20097/
...Revert to release-20161019 version, so as to revert author name from 'Pulse' to 'pulse'
10. garden2.oramap fea1a2d38c29c5cb98e7837bd63dcc8b057e9fa4 http://resource.openra.net/maps/12565/
...Revert to release-20161019 version, so as to revert map filename
11. haos_ridges_cnc.oramap c6955e257a3119df0ba9cade34cf1f87b04226dd http://resource.openra.net/maps/13118/
...Revert to release-20161019 version, so as to revert map filename and author name (Sunny_S to sunny_s)
12. Kolosseum.oramap 487b5bd2d75cee4a2985dc61393f25e38370007e http://resource.openra.net/maps/16934/
...Revert map filename and title
13. manufacturing_consent.oramap c3e27f32d460c065e4dfcc7e27d346bb75a4bd4a http://resource.openra.net/maps/20099/
...Revert to the release-20161019 version (preferred)
14. avocado.oramap f2241fd7492c326365dcbbc04745c57b18e25465 http://resource.openra.net/maps/20101/
...Revert to release-20161019 version, so as to revert author name from 'Sin' to 'sin'
15. OrderBattle4.oramap 1eeafab80d6203adacfc5a93df952114ca797a73 http://resource.openra.net/maps/19824/
...Updated to the latest version
16. PingPongBay-1.oramap c5b282f5d5bf15af88a1b0c55642b2cc0a833923 http://resource.openra.net/maps/13534/
...Revert to release-20161019 version, so as to revert author name from 'Sin' to 'sin'
17. pressure_cnc.oramap 119159eee03b084e75cdf915c5b05854ca67317f
...Revert to release-20161019 version, so as to revert map filename and author name (Sunny_S to sunny_s)
18. riversidefrontline02.oramap 120b71e205e8e86b1e9570824722586cd728b2ed http://resource.openra.net/maps/12844/
...Revert map filename and title
19. sustain.oramap 24264fd7c29e70452f79ead4b6ba13260c76bba3 http://resource.openra.net/maps/16963/
...Revert map title and author name
20. tiberian_war.oramap 084eebe3cd1161281de4ba49523ba50f8eb35194 http://resource.openra.net/maps/16970/
...Revert map filename, map title, and author name
21. toohot.oramap a536dd088a5aa394e02628d1bf1f3573ae353e03 http://resource.openra.net/maps/11902/
...Revert map filename and revert author name from 'Pulse' to 'pulse'
22. valley-of-gold-8p-update1.oramap a74b1bac24868eae1cc33b04d8560f0cac3a915c http://resource.openra.net/maps/19970/
...Updated to the latest version
23. vectorsofbattle.oramap d771d3cc80b58cd6de4306aa42c145fe73da5537 http://resource.openra.net/maps/11922/
...Revert to release-20161019 version, so as to revert map file name and author name
24. WarKraft.oramap cb676f58858abc6a866b15c85b240dc1961c67b6 http://resource.openra.net/maps/15307/
...Revert map filename


**No change**

1. antrax.oramap 45444d59551f6e97d644d1136c117f892eab0ef1 http://resource.openra.net/maps/20093/
2. Burning_Hammer.oramap 536be728bb9e1ec651f63130ab0a1b5a59e3320d http://resource.openra.net/maps/17176/
3. IN-IslandDuel.oramap 2f294c7d0d562bf9a1b0d95108e638ddfa17999e http://resource.openra.net/maps/19523/
4. kolosseum-6p.oramap b1cfad9a423851626970af3d2932b55db155526a http://resource.openra.net/maps/20098
5. morbid-aimless-poseidon.oramap 71e80a0ecfea1cd23199b2c76f8a08286086f973 http://resource.openra.net/maps/20100/
6. pirates_and_emperors.oramap 0ab25a2173c2f7639415b50d7570d6c3bb9cf8bf http://resource.openra.net/maps/13176/
7. Twisted_Peak.oramap 6bf1875739a8efa76d9bff57ae5ac41cb11c2351 http://resource.openra.net/maps/17184/


**Remove**

1. back-to-basics-8p.oramap a8aa5765acddc5679b380049642a522e37f8e2ad http://resource.openra.net/maps/13423/
2. black-tiberian-war.oramap f38a85f4f5e4d746c738dfe073562fe0be3dbe45 Deleted
3. braided-river.oramap dbbdae1f213c147557e29f05624b2ffae1055e44 Deleted
4. break-of-day.oramap 58ab529a8d12c74e44a04be2d11ba97e2d6a662d http://resource.openra.net/maps/13358/
5. chokepoint.oramap 7a4c6980577d12352a0d39aff674684749ce4eb5 http://resource.openra.net/maps/12804/
6. control-and-chaos-8p.oramap 293dbe69b6b79ef15c3442f6307ac98dea7649c6 Deleted
7. desert-storm-5.oramap b599a9110b4c68514f0c98f37dbaf9a536016c2d http://resource.openra.net/maps/15344/
8. frontal-assault.oramap d82de6e3e3139d7cbc2e62c5d65d6e04b3cc877a http://resource.openra.net/maps/16131/
9. ground-zero.oramap 54fc322c508dec12967e0ea7749eba4e0aef332f http://resource.openra.net/maps/18029/
10. guarana.oramap 647982f7e62a1dcad89d980705e012b5757f326a http://resource.openra.net/maps/16922/
11. manufacturing-consent-ns.oramap b0551b67df7c66a0805f839791453945be635612 http://resource.openra.net/maps/12006/
12. order-of-battle-10p.oramap b5c334a33e715630faebb5ab1443b06f97305699 Deleted
13. rogue-math.oramap 3b8bca51ed397cd53af8f82225ef7d5e8335295d http://resource.openra.net/maps/16949/
14. sea-and-cake.oramap b12fe2acd3ab70e10cc8217b08073723c16107eb http://resource.openra.net/maps/12168/
15. simple-chord.oramap e72c9f9b6d20a3457f1cbdc715abcc44bb9cd47f http://resource.openra.net/maps/12290/
16. the-hourglass.oramap e6759d4b609eb1e7ae89761d9cdc38082d7181c7 http://resource.openra.net/maps/12343/
17. tiberium-connected-8p.oramap 3c7e5cb39af30593fcc1a6aebcc7de516add65fd Deleted
18. tiberium-oasis-cluster.oramap 80372a6008306bd8d9afb40a2ac3cbab2e9fa761 http://resource.openra.net/maps/7317/
19. treasure-island.oramap 2b00cd4d4e200d53bcaee8f3df3fb11e3549ae9f http://resource.openra.net/maps/12100/
20. valley-of-gold-10p.oramap 715b02bf976836d6109eaa3756227485a9c7c022 Deleted
21. valley-of-gold-12p.oramap 731a5c3efec3cc531ac57390a0c161d8465a1e31 Deleted
22. valley-of-gold-2p.oramap ef9f5152f8b05c91329563d5a301d3b2d96ac333 http://resource.openra.net/maps/19939/
23. valley-of-gold-4p.oramap 0ceb3b6e807736dbb1bb72db9c665c1127bc3239 http://resource.openra.net/maps/19943/
24. valley-of-gold-6p.oramap 1cddf446aee9e3cae69d1e7b4e741a6fa5be878b http://resource.openra.net/maps/19944/
25. vectors-of-battle-10p.oramap 6e5dfb7ae10d0042a878ea4c31be35bec2bc47df Deleted
26. vectors-of-battle-8p.oramap 0ec39c602d1cc1debaed761c4adf7e2c26b41b17 Deleted

gdavegdave committed:
Update the official cnc/TD map pool

pchote commented:
Why are you reverting the map filename fixes from #12798?  I would prefer that we kept the fixed names and changed any new maps in this PR to match.

gdavegdave commented:
In some cases all that had changed were the map filenames. That caused the UID to change and so results in discontinuity with previous releases and with the Resource Center. 
Further, in a comment to the referenced pull request, GraionDilach said "You shouldn't rename map filenames since these are also used for identification on the Resource Center.".

For example,
eastwest3.oramap d07e7ade8051ce0282ea254459189ad02df3f0f6
became
east-vs-west-3.oramap 23d159326d3253b66218b8833b400b2b92ee7da2

Such changes seem pointless to me.

gdavegdave commented:
I'm working on fixing the two maps that upset Travis CI. After that, I'll set about reverting the filenames to the playtest-20170304 names (for the common maps) and to change the new maps to match. Please let me know if that's still what you'd like to happen.

pchote commented:
The UID is calculated from the *contents* of the map package, so changing the filename does not change the UID:

```
paul@Ragnarok:OpenRA+master$ mono OpenRA.Utility.exe cnc --map-hash mods/cnc/maps/east-vs-west-3.oramap 
23d159326d3253b66218b8833b400b2b92ee7da2
paul@Ragnarok:OpenRA+master$ mv mods/cnc/maps/east-vs-west-3.oramap mods/cnc/maps/eastwest3.oramap 
paul@Ragnarok:OpenRA+master$ mono OpenRA.Utility.exe cnc --map-hash mods/cnc/maps/eastwest3.oramap 
23d159326d3253b66218b8833b400b2b92ee7da2
```
You must have made some other changes to the map (loading and resaving in the map editor would be enough) to change the UID.

Similarly, the resource center doesn't care about filenames.  It assigns each map its own numeric id, which is what you see in the URLs, as well as using UIDs.

reaperrr commented:
> You must have made some other changes to the map *(loading and resaving in the map editor would be enough)* to change the UID.

That's what @dtluna might have done in his PR, though, so reverting to previous file versions might indeed help if that was the case.

pchote commented:
Reverting is fine, so long as they are then renamed back to the better name.

SoScared commented:
Author names on:

16:9 - Your name here
Contra - % - Norman?
Lost Souls - Norman
Model 200 - symbols - Norman?
Model 150 - Norman
Tiberian War - Anonymous
Alpha Juno V - Norman

Norman loves to use all kinds of author names and name styles. Should stick with 1 for the pool.

gdavegdave commented:
I've endeavoured to ensure that we have the best version for each map. Also, I've cleaned up the file names, map names and author names (including unifying norman!). I'll update the pull request after a third pair of eyes have checked my changes.

dtluna commented:
I don't understand why @gdavegdave has removed so many maps I've added, and added many choky maps like "Crossing The Divide".

gdavegdave commented:
I consider this PR now complete. Please review it and let me know if any changes are required. I have sanitised the map file names, map titles and author names (i.e. given maps by norman a single author name). I include below a list of all the maps along with their UIDs and links to the resource centre. For those maps that were changed slightly (typically map title or author), I uploaded the new version to the Resource Center. As such, the stats for those maps were reset.

For convenience, here are archives of the maps:
[maps.tar.gz](https://github.com/OpenRA/OpenRA/files/870195/maps.tar.gz)
[maps.zip](https://github.com/OpenRA/OpenRA/files/870196/maps.zip)

**counter** **map_filename** **map_UID** **map_link**
1. 16-9.oramap ca7dc1e8cf61bc531a503d72d95c993ac8fdeee4 http://resource.openra.net/maps/20117/
2. aggressive-tendencies.oramap c39e94b8d9fb1f733a2c39af93e388649ed8d188 http://resource.openra.net/maps/13148/
3. alpha-juno-5.oramap cafa8747acd079aee56042a2346130294bc9ec0c http://resource.openra.net/maps/20118/
4. antrax.oramap 971052f825eba9e3be0419ac4a68074eecff48a5 http://resource.openra.net/maps/20119/
5. arcade.oramap 5c3b6026773e4401aa443bcd1f753db721a9a396 http://resource.openra.net/maps/20120/
6. avocado.oramap f2241fd7492c326365dcbbc04745c57b18e25465 http://resource.openra.net/maps/20121/
7. axis-of-advance.oramap 2e3bffcd1a4b946447c1910eea96ac5257173c76 http://resource.openra.net/maps/20122/
8. back-to-basics.oramap bc3b69c25bf57ea0738bd774366f7634a939305f http://resource.openra.net/maps/13423/
9. blood-and-sand.oramap b4a8ea0701db2874f3a7a5a50631617f0ecf99b2 http://resource.openra.net/maps/12728/
10. botany.oramap dbaa2606988ff9e3cbefd9a8572c946d7919538b http://resource.openra.net/maps/11906/
11. burning-hammer.oramap 536be728bb9e1ec651f63130ab0a1b5a59e3320d http://resource.openra.net/maps/17176/
12. carters-ridge.oramap 7973ceac3053a3568a5746e1cca5d251d07d3059 http://resource.openra.net/maps/17571/
13. chill-out.oramap 8692c62757df9d5bf3465937258a330cdd0e2466 http://resource.openra.net/maps/11901/
14. collateral-chaos.oramap 4a4287267b15ede75b4387515c9f26950482c6b6 http://resource.openra.net/maps/19656/
15. contra.oramap b13ca66a03f6f8a0e4a4145c1924659dab032ffe http://resource.openra.net/maps/20123/
16. control-and-chaos.oramap 449308b7191bce0e74f4adca69f12453abd80e6e http://resource.openra.net/maps/13265/
17. crossing-the-divide.oramap f46588829b7dca039cbb51b7c351fbcd128ba746 http://resource.openra.net/maps/17425/
18. dead-in-motion.oramap f49c1ee505f67e960a11fd1bf3f4d08bf46e2241 http://resource.openra.net/maps/20124/
19. desert-lakes.oramap 5d7ddbabb602473251c0fcccc35d952f199953bc http://resource.openra.net/maps/20125/
20. desert-rats-cnc.oramap 1717f536d90a6e1f79a5e25c8f1bd83835ac8eaf http://resource.openra.net/maps/20126/
21. desert-storm-3.oramap b0253ecf36ad81c54e7db89d1ec6f5e45f711025 http://resource.openra.net/maps/20127/
22. deterring-democracy-plus.oramap fb10704c557f17787a97c16d63c9143a1a387f14 http://resource.openra.net/maps/12608/
23. deterring-democracy.oramap 90391ffab4b4701f90b29bf4058276beb6576b5f http://resource.openra.net/maps/13237/
24. dualism.oramap 51853b5c481eae2a487b2909ee35aaa3d86f3d18 http://resource.openra.net/maps/11915/
25. east-vs-west-3.oramap d07e7ade8051ce0282ea254459189ad02df3f0f6 http://resource.openra.net/maps/12400/
26. escalations.oramap 3468a2fc1e4387cfd412f0b54ced46c0de76c068 http://resource.openra.net/maps/20128/
27. fortresses-3.oramap df74b7a09a1933fb5fa8ceb6e94660ed100c4624 http://resource.openra.net/maps/20129/
28. frigid-pass.oramap 96fee4a172f9c44a3034bd79f95424c1773e9c15 http://resource.openra.net/maps/17910/
29. frostways.oramap 7dd88799c41e61e5ff0a670203ae9b7db26e2f96 http://resource.openra.net/maps/20130/
30. haos-ridges-cnc.oramap c6955e257a3119df0ba9cade34cf1f87b04226dd http://resource.openra.net/maps/13118/
31. hegemony-or-survival.oramap 45a72f7370681317cc1256041058f8012abdc187 http://resource.openra.net/maps/20131/
32. island-duel.oramap 2f294c7d0d562bf9a1b0d95108e638ddfa17999e http://resource.openra.net/maps/19523/
33. jungle-conflict.oramap 5d5f51a0cee29cdd4c104718994cd0170b7d3382 http://resource.openra.net/maps/13448/
34. kolosseum.oramap 94b1eda23b5fc3fa19c7d3bceefe71ca90c44a4d http://resource.openra.net/maps/20132/
35. letters-from-lexington.oramap 291e3707fca0c98bbbd6acccb180a23882717a49 http://resource.openra.net/maps/14560/
36. lost-souls.oramap d679df3037b2726ea913dcd424397d7bcfd8397a http://resource.openra.net/maps/20133/
37. manufacturing-consent.oramap c3e27f32d460c065e4dfcc7e27d346bb75a4bd4a http://resource.openra.net/maps/20134/
38. media-control.oramap 136a63ef09de790f5a062284078e02f936fa51f6 http://resource.openra.net/maps/20135/
39. model-150.oramap 3606f40cb5e52aa7d6f8a8bcf0236f36fc9b9eca http://resource.openra.net/maps/20136/
40. model-200.oramap e2e21c6c34fc07cde41f8d9c7f70b06d9c8006df http://resource.openra.net/maps/20137/
41. morbid-aimless-poseidon.oramap 71e80a0ecfea1cd23199b2c76f8a08286086f973 http://resource.openra.net/maps/20138/
42. mt-nukebait.oramap a1111dda522c0e498dd39296e30460d013c70a4b http://resource.openra.net/maps/20139/
43. order-of-battle.oramap 1eeafab80d6203adacfc5a93df952114ca797a73 http://resource.openra.net/maps/19824/
44. ping-pong-bay.oramap c5b282f5d5bf15af88a1b0c55642b2cc0a833923 http://resource.openra.net/maps/13534/
45. pirates-and-emperors.oramap 0ab25a2173c2f7639415b50d7570d6c3bb9cf8bf http://resource.openra.net/maps/13176/
46. plan-b.oramap 8855d061c65f6972194ee46efbdc986dcf39c32d http://resource.openra.net/maps/20140/
47. pressure-cnc.oramap 119159eee03b084e75cdf915c5b05854ca67317f http://resource.openra.net/maps/12809/
48. reasonable-doubt.oramap b6cefe97e71a9015876c395177516e2d237e1510 http://resource.openra.net/maps/12615/
49. riverside-frontline.oramap 6216f498cdf9e9e9db87415bb4bcf0972d1073e9 http://resource.openra.net/maps/20141/
50. scorched-earth.oramap c9bdfd8f71a96472aa7b06d124ad3cdcd9b1b6d8 http://resource.openra.net/maps/18504/
51. sustain.oramap 9f154c4358fead604dfd30ecd02203187ad7e2be http://resource.openra.net/maps/20142/
52. tactical-advance.oramap e75c392ff23c327c25b61224b1524a649d893618 http://resource.openra.net/maps/17483/
53. thawed-front.oramap f561aaea8ce10996d118aa3e4854a461f2e628d3 http://resource.openra.net/maps/18035/
54. the-hot-box.oramap 7f6de317db131e2051d67ffe4ae300df3adf2656 http://resource.openra.net/maps/20150/
55. the-new-mandarins.oramap 2b893052f94fc79dfb64320fff7742e684361222 http://resource.openra.net/maps/20151/
56. tiberian-falls.oramap ee94bc34598b56237c7a42ec2a6d3be2d5ea002a http://resource.openra.net/maps/20034/
57. tiberian-war.oramap 86fa2c7ad5d1bb46612b971ad7356013b6ce31d5 http://resource.openra.net/maps/20152/
58. tiberium-canopy.oramap dab8fefee5db7e71c65083f77aa198f0673ab621 http://resource.openra.net/maps/18270/
59. tiberium-garden.oramap fea1a2d38c29c5cb98e7837bd63dcc8b057e9fa4 http://resource.openra.net/maps/12565/
60. too-hot.oramap a536dd088a5aa394e02628d1bf1f3573ae353e03 http://resource.openra.net/maps/11902/
61. trial-of-possession.oramap a8630d3836b63e05c8f981027f49006375725f59 http://resource.openra.net/maps/17198/
62. twin-lakes.oramap d5d6262e6676b7a4168a929fd6bebc34226ea40e http://resource.openra.net/maps/17426/
63. twisted-peak.oramap 6bf1875739a8efa76d9bff57ae5ac41cb11c2351 http://resource.openra.net/maps/17184/
64. two-ponds.oramap d5b1bfa40ebfc093ac29c4d7eced8e1dcbcee573 http://resource.openra.net/maps/13371/
65. understanding-power.oramap e2c98e5eea32f4f206b5661c75e9c6a82c902339 http://resource.openra.net/maps/11824/
66. valley-of-gold.oramap 45c741a4c83ed348ab4318e250249f6a8957401c http://resource.openra.net/maps/20153/
67. vectors-of-battle.oramap d771d3cc80b58cd6de4306aa42c145fe73da5537 http://resource.openra.net/maps/20154/
68. warchild.oramap 95f2e7fc4e36e60be4c0aa64b63cb45bfb546f53 http://resource.openra.net/maps/17178/
69. warkraft.oramap 112f9c05512615c11306a9365fd48e92ea949000 http://resource.openra.net/maps/20155/
70. when-two-people-quarrel.oramap 8c92bac298229929ae82848c89dfcc0a1ca93a7b http://resource.openra.net/maps/20156/
71. winters-end.oramap 3a1ce5ceea6f7dfc564ede5b86aaf3a5c45a274b http://resource.openra.net/maps/20157/
72. x896.oramap 084b4bddd86b79671048f3fd289e3c68c1ce4886 http://resource.openra.net/maps/13413/
73. year-501.oramap d7f75c0b952e88309c4dc1ba2c80ab5cff5df720 http://resource.openra.net/maps/20158/

reaperrr commented:
Quoting @pchote from #12798:
> Please add back Tiberian Garden and *Chokepoint*. I use these for testing the AI, and they're among the most popular shipped maps.

So please let's keep it (I'd prefer to keep it myself, too).

Also, I agree with Sire from the forum and would like to keep Break of Day around for nostalgy/aesthetics/uniqueness/AI+pathfinder testing.

Other than that, I'll trust the community and 👍 this once these 2 are back.

pchote commented:
Two more minor issues:
* Please squash the fixup commits back into the main ones.
* All maps that include custom rules (like "the hot box") *must* be unpacked.

I still haven't had a chance to look at the maps themselves, but still plan to.

gdavegdave commented:
@reaperrr: I gladly added back Chokepoint and Break of Day.

@pchote: I unpacked The Hot Box and I verified that I had removed the custom rules from the other maps (e.g. the observation post from Media Control and the GlobalLightingPaletteEffect from Desert Rats). Further, I squashed the commits into a single commit.

Below is an up to date list of all the maps. Here are the usual map archives:
[maps.tar.gz](https://github.com/OpenRA/OpenRA/files/873741/maps.tar.gz)
[maps.zip](https://github.com/OpenRA/OpenRA/files/873742/maps.zip)

**counter** **map_filename** **map_UID** **map_link**
1. 16-9.oramap ca7dc1e8cf61bc531a503d72d95c993ac8fdeee4 http://resource.openra.net/maps/20117/
2. aggressive-tendencies.oramap c39e94b8d9fb1f733a2c39af93e388649ed8d188 http://resource.openra.net/maps/13148/
3. alpha-juno-5.oramap cafa8747acd079aee56042a2346130294bc9ec0c http://resource.openra.net/maps/20118/
4. antrax.oramap 971052f825eba9e3be0419ac4a68074eecff48a5 http://resource.openra.net/maps/20119/
5. arcade.oramap 5c3b6026773e4401aa443bcd1f753db721a9a396 http://resource.openra.net/maps/20120/
6. avocado.oramap f2241fd7492c326365dcbbc04745c57b18e25465 http://resource.openra.net/maps/20121/
7. axis-of-advance.oramap 2e3bffcd1a4b946447c1910eea96ac5257173c76 http://resource.openra.net/maps/20122/
8. back-to-basics.oramap bc3b69c25bf57ea0738bd774366f7634a939305f http://resource.openra.net/maps/13423/
9. blood-and-sand.oramap b4a8ea0701db2874f3a7a5a50631617f0ecf99b2 http://resource.openra.net/maps/12728/
10. botany.oramap dbaa2606988ff9e3cbefd9a8572c946d7919538b http://resource.openra.net/maps/11906/
11. break-of-day.oramap 58ab529a8d12c74e44a04be2d11ba97e2d6a662d http://resource.openra.net/maps/13358/
12. burning-hammer.oramap 536be728bb9e1ec651f63130ab0a1b5a59e3320d http://resource.openra.net/maps/17176/
13. carters-ridge.oramap 7973ceac3053a3568a5746e1cca5d251d07d3059 http://resource.openra.net/maps/17571/
14. chill-out.oramap 8692c62757df9d5bf3465937258a330cdd0e2466 http://resource.openra.net/maps/11901/
15. chokepoint.oramap 7a4c6980577d12352a0d39aff674684749ce4eb5 http://resource.openra.net/maps/12804/
16. collateral-chaos.oramap 4a4287267b15ede75b4387515c9f26950482c6b6 http://resource.openra.net/maps/19656/
17. contra.oramap b13ca66a03f6f8a0e4a4145c1924659dab032ffe http://resource.openra.net/maps/20123/
18. control-and-chaos.oramap 449308b7191bce0e74f4adca69f12453abd80e6e http://resource.openra.net/maps/13265/
19. crossing-the-divide.oramap f46588829b7dca039cbb51b7c351fbcd128ba746 http://resource.openra.net/maps/17425/
20. dead-in-motion.oramap f49c1ee505f67e960a11fd1bf3f4d08bf46e2241 http://resource.openra.net/maps/20124/
21. desert-lakes.oramap 5d7ddbabb602473251c0fcccc35d952f199953bc http://resource.openra.net/maps/20125/
22. desert-rats-cnc.oramap 1717f536d90a6e1f79a5e25c8f1bd83835ac8eaf http://resource.openra.net/maps/20126/
23. desert-storm-3.oramap b0253ecf36ad81c54e7db89d1ec6f5e45f711025 http://resource.openra.net/maps/20127/
24. deterring-democracy-plus.oramap fb10704c557f17787a97c16d63c9143a1a387f14 http://resource.openra.net/maps/12608/
25. deterring-democracy.oramap 90391ffab4b4701f90b29bf4058276beb6576b5f http://resource.openra.net/maps/13237/
26. dualism.oramap 51853b5c481eae2a487b2909ee35aaa3d86f3d18 http://resource.openra.net/maps/11915/
27. east-vs-west-3.oramap d07e7ade8051ce0282ea254459189ad02df3f0f6 http://resource.openra.net/maps/12400/
28. escalations.oramap 3468a2fc1e4387cfd412f0b54ced46c0de76c068 http://resource.openra.net/maps/20128/
29. fortresses-3.oramap df74b7a09a1933fb5fa8ceb6e94660ed100c4624 http://resource.openra.net/maps/20129/
30. frigid-pass.oramap 96fee4a172f9c44a3034bd79f95424c1773e9c15 http://resource.openra.net/maps/17910/
31. frostways.oramap 7dd88799c41e61e5ff0a670203ae9b7db26e2f96 http://resource.openra.net/maps/20130/
32. haos-ridges-cnc.oramap c6955e257a3119df0ba9cade34cf1f87b04226dd http://resource.openra.net/maps/13118/
33. hegemony-or-survival.oramap 45a72f7370681317cc1256041058f8012abdc187 http://resource.openra.net/maps/20131/
34. island-duel.oramap 2f294c7d0d562bf9a1b0d95108e638ddfa17999e http://resource.openra.net/maps/19523/
35. jungle-conflict.oramap 5d5f51a0cee29cdd4c104718994cd0170b7d3382 http://resource.openra.net/maps/13448/
36. kolosseum.oramap 94b1eda23b5fc3fa19c7d3bceefe71ca90c44a4d http://resource.openra.net/maps/20132/
37. letters-from-lexington.oramap 291e3707fca0c98bbbd6acccb180a23882717a49 http://resource.openra.net/maps/14560/
38. lost-souls.oramap d679df3037b2726ea913dcd424397d7bcfd8397a http://resource.openra.net/maps/20133/
39. manufacturing-consent.oramap c3e27f32d460c065e4dfcc7e27d346bb75a4bd4a http://resource.openra.net/maps/20134/
40. media-control.oramap 136a63ef09de790f5a062284078e02f936fa51f6 http://resource.openra.net/maps/20135/
41. model-150.oramap 3606f40cb5e52aa7d6f8a8bcf0236f36fc9b9eca http://resource.openra.net/maps/20136/
42. model-200.oramap e2e21c6c34fc07cde41f8d9c7f70b06d9c8006df http://resource.openra.net/maps/20137/
43. morbid-aimless-poseidon.oramap 71e80a0ecfea1cd23199b2c76f8a08286086f973 http://resource.openra.net/maps/20138/
44. mt-nukebait.oramap a1111dda522c0e498dd39296e30460d013c70a4b http://resource.openra.net/maps/20139/
45. order-of-battle.oramap 1eeafab80d6203adacfc5a93df952114ca797a73 http://resource.openra.net/maps/19824/
46. ping-pong-bay.oramap c5b282f5d5bf15af88a1b0c55642b2cc0a833923 http://resource.openra.net/maps/13534/
47. pirates-and-emperors.oramap 0ab25a2173c2f7639415b50d7570d6c3bb9cf8bf http://resource.openra.net/maps/13176/
48. plan-b.oramap 8855d061c65f6972194ee46efbdc986dcf39c32d http://resource.openra.net/maps/20140/
49. pressure-cnc.oramap 119159eee03b084e75cdf915c5b05854ca67317f http://resource.openra.net/maps/12809/
50. reasonable-doubt.oramap b6cefe97e71a9015876c395177516e2d237e1510 http://resource.openra.net/maps/12615/
51. riverside-frontline.oramap 6216f498cdf9e9e9db87415bb4bcf0972d1073e9 http://resource.openra.net/maps/20141/
52. scorched-earth.oramap c9bdfd8f71a96472aa7b06d124ad3cdcd9b1b6d8 http://resource.openra.net/maps/18504/
53. sustain.oramap 9f154c4358fead604dfd30ecd02203187ad7e2be http://resource.openra.net/maps/20142/
54. tactical-advance.oramap e75c392ff23c327c25b61224b1524a649d893618 http://resource.openra.net/maps/17483/
55. thawed-front.oramap f561aaea8ce10996d118aa3e4854a461f2e628d3 http://resource.openra.net/maps/18035/
56. the-hot-box 7f6de317db131e2051d67ffe4ae300df3adf2656 http://resource.openra.net/maps/20150/
57. the-new-mandarins.oramap 2b893052f94fc79dfb64320fff7742e684361222 http://resource.openra.net/maps/20151/
58. tiberian-falls.oramap ee94bc34598b56237c7a42ec2a6d3be2d5ea002a http://resource.openra.net/maps/20034/
59. tiberian-war.oramap 86fa2c7ad5d1bb46612b971ad7356013b6ce31d5 http://resource.openra.net/maps/20152/
60. tiberium-canopy.oramap dab8fefee5db7e71c65083f77aa198f0673ab621 http://resource.openra.net/maps/18270/
61. tiberium-garden.oramap fea1a2d38c29c5cb98e7837bd63dcc8b057e9fa4 http://resource.openra.net/maps/12565/
62. too-hot.oramap a536dd088a5aa394e02628d1bf1f3573ae353e03 http://resource.openra.net/maps/11902/
63. trial-of-possession.oramap a8630d3836b63e05c8f981027f49006375725f59 http://resource.openra.net/maps/17198/
64. twin-lakes.oramap d5d6262e6676b7a4168a929fd6bebc34226ea40e http://resource.openra.net/maps/17426/
65. twisted-peak.oramap 6bf1875739a8efa76d9bff57ae5ac41cb11c2351 http://resource.openra.net/maps/17184/
66. two-ponds.oramap d5b1bfa40ebfc093ac29c4d7eced8e1dcbcee573 http://resource.openra.net/maps/13371/
67. understanding-power.oramap e2c98e5eea32f4f206b5661c75e9c6a82c902339 http://resource.openra.net/maps/11824/
68. valley-of-gold.oramap 45c741a4c83ed348ab4318e250249f6a8957401c http://resource.openra.net/maps/20153/
69. vectors-of-battle.oramap d771d3cc80b58cd6de4306aa42c145fe73da5537 http://resource.openra.net/maps/20154/
70. warchild.oramap 95f2e7fc4e36e60be4c0aa64b63cb45bfb546f53 http://resource.openra.net/maps/17178/
71. warkraft.oramap 112f9c05512615c11306a9365fd48e92ea949000 http://resource.openra.net/maps/20155/
72. when-two-people-quarrel.oramap 8c92bac298229929ae82848c89dfcc0a1ca93a7b http://resource.openra.net/maps/20156/
73. winters-end.oramap 3a1ce5ceea6f7dfc564ede5b86aaf3a5c45a274b http://resource.openra.net/maps/20157/
74. x896.oramap 084b4bddd86b79671048f3fd289e3c68c1ce4886 http://resource.openra.net/maps/13413/
75. year-501.oramap d7f75c0b952e88309c4dc1ba2c80ab5cff5df720 http://resource.openra.net/maps/20158/

Irnub commented:
Hey guys. See below, what should be the final upload for Order of Battle. I had accidentally deleted a few trees near the spawns which made some tree areas passable when they were not supposed to be. This is fixed here - http://resource.openra.net/maps/20223/

gdavegdave commented:
@Irnub I have amended the PR and edited my last post to account for your updated Order of Battle (the map file, the entry in the list and the map archives).

GraionDilach commented:
Please restore the map lighting on Desert Rats. That feature of OpenRA's is still heavily underutilized while it adds a lot to a map's atmosphere.

gdavegdave commented:
@GraionDilach I restored the map lighting to Desert Rats. 

I had mistakenly thought that maps (other than  The Hot Box) could not use custom rules. I now understand that custom rules using official features, such as GlobalLightingPaletteEffect, are permitted, whilst unofficial mods such as observation posts and re-capturable oil derricks are not. 

I updated the list above with the correct UID and link for desert-rats-cnc.oramap and I updated maps.zip and maps.tar.gz: https://github.com/OpenRA/OpenRA/pull/13019#issuecomment-289580017.

gdavegdave commented:
@reaperrr I just noticed your polish of Morbid Aimless Poseidon: https://github.com/OpenRA/OpenRA/pull/12908

Should I update this PR to include your map, or will you be able to prefer your version in a selective merge?

reaperrr commented:
@gdavegdave I think it would be easier if you updated your PR.

gdavegdave commented:
@reaperrr I've updated my PR with the polished Morbid Aimless Poseidon. 

I've updated the UID and link, along with the archives, in the table above to reflect this (https://github.com/OpenRA/OpenRA/pull/13019#issuecomment-289580017).

pchote commented:
These look pretty good overall: a good level of polish overall, what looks like a decent mix of map types, and strong support from the players on the forum.  I wouldn't have minded if you'd left out some of old maps that @dtluna removed in his first PR, but i'm also not *too* fussed about adding them back (we can always remove them in a future release after the playerbase adjusts).

Here's some comments on specific maps:

When two people quarrel, Fortresses III:
* Obvious tiling errors in the big muddy river crossings.  Can we fix or remove these?

Chill out:
* Ugly tiling errors at the ends of the rivers when the trees are burnt by fire or superweapons.

Crossing the divide, Frostways, Winter's End:
* Boy, the map authors sure like their smudge tiles... IMO these maps would gain a lot more polish by toning these down.

Most norman maps:
* Please change his name to 'norman'.  Using weird unicode characters make it difficult to filter maps by his name, and hurts readability.
* Many of his maps feature destroyed oil derricks, which I assume were there to be repaired until the custom rule set.  They aren't useful in the default rule set, so may want to be replaced with living ones?
* Why does he put flares next to oil derricks on so many maps?

@dtluna: do you have any other thoughts or comments about this PR?

ltem commented:
Till now I looked over all the two player maps and found following issues
- Pirates & Emperors: The tibtrees need to be moved down by one tile and I think that the two blue tib trees in the "middle" of the map (see image as reference for one of them) are not equal because they don't have the same are to spread, maybe that this can be modified as well.
![image](https://cloud.githubusercontent.com/assets/20945684/24563570/1ad71654-164f-11e7-9a12-957be23946f8.png)
- Two ponds: Same issue as above, tibtrees one tile down
- The New Mandarins has some tiling errors on the island in the middle of the map (the tiles are from the river set, not sure if this can be fixed or is wanted)
![image](https://cloud.githubusercontent.com/assets/20945684/24564388/fa52dec4-1651-11e7-95ee-a37effeb5419.png)
- Understanding Power has a issue at position 22, 11
![image](https://cloud.githubusercontent.com/assets/20945684/24564451/3ab5aa78-1652-11e7-94f8-d7386de71dd3.png)

gdavegdave commented:
I anticipate that the requested changes (and more) will be complete and uploaded by today or tomorrow.

gdavegdave commented:
Thank you to both @pchote and @ltem  for reviewing the map selection and highlighting tile errors, etc. In the last few days, The Echo of Damnation ('Echo'; a major OpenRA map maker) and I have worked through the requested changes. I performed the admin and fixed the minor issues, (moving tib trees down 1 cell and fixing author names). Echo took care of all of the complicated stuff, going way beyond what was requested in the things that he identified and fixed.

Here is a summary of all of the changes:

1. 16-9.oramap - Restored 2 oil derricks and changed map author to 'norman'.
2. alpha-juno-5.oramap - Changed map author to 'norman'.
3. antrax.oramap - Changed map author to 'norman'.
4. arcade.oramap - Changed map author to 'norman'.
5. blood-and-sand.oramap -  Fixed numerous beach and cliff tiling errors throughout and slightly modified SW corner of map.
6. break-of-day.oramap - Moved tib trees down 1 cell.
7. chill-out.oramap - Fixed tile errors at river ends by running the river into two pools and altered the cliffing and trees to achieve this.
8. chokepoint.oramap - Moved tib trees down 1 cell.
9. contra.oramap - Restored 2 oil derricks and changed map author to 'norman'.
10. crossing-the-divide.oramap - Reduced smudge tiles.
11. dead-in-motion.oramap - Moved tib trees down 1 cell.
12. desert-storm-3.oramap - Restored 4 oil derricks (2 accessible only by air) and changed map author to 'norman'.
13. deterring-democracy.oramap - Moved tib trees down 1 cell.
14. deterring-democracy-plus.oramap - Moved tib trees down 1 cell. Fixed cliff issues. Modified NW & NE Beaches to fix squared off edges, glitches and missing pieces. Fixed south island beach on eastern side.
15. east-vs-west-3.oramap - Moved tib trees down 1 cell.
16. fortresses-3.oramap - Replaced by @irnub's tropic-expanse.oramap (a 4 player FFA).
17. frostways.oramap - Reduced smudge tiles.
18. haos-ridges-cnc.oramap - Moved tib trees down 1 cell.
19. hegemony-or-survival.oramap - Moved tib trees down 1 cell.
20. kolosseum.oramap - Changed map author to 'norman'.
21. lost-souls_modded.oramap - Changed map author to 'norman'.
22. manufacturing-consent.oramap - Moved tib trees down 1 cell. Fixed cliff errors, trying to stick to the original shapes, but changed a few.
23. model-150.oramap - Fixed tile errors at river ends by placing rivers leading into pools; changed map author to 'norman'.
24. model-200.oramap - Changed map author to 'norman'.
25. morbid-aimless-poseidon.oramap - Fixed aforementioned tile errors and some other beach and cliff tiles. Found other errors but these are not fixable given the limitations of the tileset and the map construction.
26. pirates-and-emperors.oramap - Moved tib trees down 1 cell and minor eco balancing. Fixed SW river and pond, eastern ponds and NE ponds, as well as some minor cliff glitches.
27. plan-b.oramap - Changed map author to 'norman'.
28. pressure-cnc.oramap - Fixed multiple beach and cliff errors. Removed the river and replaced with a merged cliff. Removed and added trees and debris in some areas.
29. sustain.oramap - Changed map author to 'norman'.
30. the-new-mandarins.oramap - Fixed a few cliff and beach errors.
31. tiberian-war.oramap - Changed map author to 'norman'.
32. tiberium-garden-2.oramap - Moved tib trees down 1 cell. 
33. two-ponds.oramap - Moved tib trees down 1 cell. Fixed river, beach and cliff glitches.
34. understanding-power.oramap - Fixed identified issue, as well as numerous cliff issues and modified the beach pools on the N&S, which also required minor adjacent cliff edits.
35. warchild.oramap - Fixed the beach tiles at the aforementioned locations.
36. warkraft.oramap - Restored 6 oil derricks and changed map author to 'norman'.
37. when-two-people-quarrel.oramap - replaced by @irnub's hattrix.oramap (also a 3 player FFA)
38. winters-end.oramap - the effect achieved by the use of the smudge tiles is felt to be integral to the map.

The limitations of the current tileset meant it was not feasible to fix the tile errors on fortresses-3.oramap and when-two-people-quarrel.oramap without reducing the width of the crossings and thus changing the gameplay, so both maps were replaced.

I think that norman places flares next to some oil derricks for three reasons: (1) they can be used to hide a minigunner to e.g. snipe engis (2) they highlight the derrick's existence when it is reachable only by air (3) aesthetics

Of the maps that dtluna had removed, we had only added back 5 maps, each popular: Dead in Motion (1v1), Two Ponds (1v1), Deterring Democracy + (2v2), Mt. Nukebait (2v2; a unique map), and Hegemony or Survival (4v4).


Below is an up to date summary of all of the maps and here are the usual archives:
[maps.tar.gz](https://github.com/OpenRA/OpenRA/files/893922/maps.tar.gz)
[maps.zip](https://github.com/OpenRA/OpenRA/files/893923/maps.zip)


**counter** **map_filename** **map_UID** **map_link**
1. 16-9.oramap dbce6538cdfe130e0f7a350a63510ea41f4b1a12 http://resource.openra.net/maps/20371/
2. aggressive-tendencies.oramap c39e94b8d9fb1f733a2c39af93e388649ed8d188 http://resource.openra.net/maps/13148/
3. alpha-juno-5.oramap e306b0b6ee532a74eb9ee887e8b83e1586838629 http://resource.openra.net/maps/20372/
4. antrax.oramap c6dcbc855b6a70c5ee6509d614c2d8e0bb5fd970 http://resource.openra.net/maps/20373/
5. arcade.oramap 997e8cb6c3c53682d88773e1aac6299868c0e384 http://resource.openra.net/maps/20405/
6. avocado.oramap f2241fd7492c326365dcbbc04745c57b18e25465 http://resource.openra.net/maps/20121/
7. axis-of-advance.oramap 2e3bffcd1a4b946447c1910eea96ac5257173c76 http://resource.openra.net/maps/20122/
8. back-to-basics.oramap bc3b69c25bf57ea0738bd774366f7634a939305f http://resource.openra.net/maps/13423/
9. blood-and-sand.oramap f710b3fe8ef74df19e0a8f14a03916ef8fd55833 http://resource.openra.net/maps/20374
10. botany.oramap dbaa2606988ff9e3cbefd9a8572c946d7919538b http://resource.openra.net/maps/11906/
11. break-of-day.oramap 5fc52741df680d656e6713d25c7a645a2de5b70f http://resource.openra.net/maps/20375/
12. burning-hammer.oramap 536be728bb9e1ec651f63130ab0a1b5a59e3320d http://resource.openra.net/maps/17176/
13. carters-ridge.oramap 7973ceac3053a3568a5746e1cca5d251d07d3059 http://resource.openra.net/maps/17571/
14. chill-out.oramap b950233cbb3bcfefc74c72b1cc7d99b7ac159ab3 http://resource.openra.net/maps/20406/
15. chokepoint.oramap 1008cb4cae437f07f13866a85b1609157d3200a8 http://resource.openra.net/maps/20376/
16. collateral-chaos.oramap 4a4287267b15ede75b4387515c9f26950482c6b6 http://resource.openra.net/maps/19656/
17. contra.oramap ab951969ee6c234b15c9f11a74dca4ec63e64f85 http://resource.openra.net/maps/20377/
18. control-and-chaos.oramap 449308b7191bce0e74f4adca69f12453abd80e6e http://resource.openra.net/maps/13265/
19. crossing-the-divide.oramap 2a351aa1265af0afde823aa7775cf68116941170 http://resource.openra.net/maps/20308/
20. dead-in-motion.oramap 0d4845d4c61ee06c95f394126c913c365072f108 http://resource.openra.net/maps/20378/
21. desert-lakes.oramap 5d7ddbabb602473251c0fcccc35d952f199953bc http://resource.openra.net/maps/20125/
22. desert-rats-cnc 0cfda3ee9cda9259604eb970193bfbf999ad364c http://resource.openra.net/maps/20225/
23. desert-storm-3.oramap 52aa302abf8a094194711958b41f9a6eec1a263b http://resource.openra.net/maps/20379/
24. deterring-democracy-plus.oramap b5bfd7d220923d64cd41892acf6b94c7fd56f10c http://resource.openra.net/maps/20381/
25. deterring-democracy.oramap d02a4e11ec802f8cd791c823e22488800ce27afb http://resource.openra.net/maps/20380/
26. dualism.oramap 51853b5c481eae2a487b2909ee35aaa3d86f3d18 http://resource.openra.net/maps/11915/
27. east-vs-west-3.oramap d2f2a2a715e6d42dac77d37db6bcf6a05efc380c http://resource.openra.net/maps/20382/
28. escalations.oramap 3468a2fc1e4387cfd412f0b54ced46c0de76c068 http://resource.openra.net/maps/20128/
29. frigid-pass.oramap 96fee4a172f9c44a3034bd79f95424c1773e9c15 http://resource.openra.net/maps/17910/
30. frostways.oramap f0d4a52915d472b566fbe2f795c5f83e2af7461c http://resource.openra.net/maps/20383/
31. haos-ridges-cnc.oramap 484ff83930492fb2bf9b6603b2615a02c85ed25a http://resource.openra.net/maps/20384/
32. hattrix acebd51bb41de8f7bf83c9afa0b742f72b10164e http://resource.openra.net/maps/20357/
33. hegemony-or-survival.oramap cbf6f949f246734f0e3ba5111d48f7bb7f93c2d9 http://resource.openra.net/maps/20385/
34. island-duel.oramap 2f294c7d0d562bf9a1b0d95108e638ddfa17999e http://resource.openra.net/maps/19523/
35. jungle-conflict.oramap 5d5f51a0cee29cdd4c104718994cd0170b7d3382 http://resource.openra.net/maps/13448/
36. kolosseum.oramap b98fdf950a718c03d304dfd05404526243eed4a2 http://resource.openra.net/maps/20386/
37. letters-from-lexington.oramap 291e3707fca0c98bbbd6acccb180a23882717a49 http://resource.openra.net/maps/14560/
38. lost-souls.oramap 0bcc6a3fe46741575943f98abba7026cb1692609 http://resource.openra.net/maps/20387/
39. manufacturing-consent.oramap 04baa303f8f3c3f30c39c60e02ae522a89a1ea33 http://resource.openra.net/maps/20388/
40. media-control.oramap 136a63ef09de790f5a062284078e02f936fa51f6 http://resource.openra.net/maps/20135/
41. model-150.oramap 293cdf1766a282aa8747bbec99b1abb7e2f5b928 http://resource.openra.net/maps/20389/
42. model-200.oramap 165d81039ba83295552b81dcd5a0b8fdd4295d85 http://resource.openra.net/maps/20390/
43. morbid-aimless-poseidon.oramap 12b6da034d22e624e81eb2b6ed7ff66f894b03ba http://resource.openra.net/maps/20393/
44. mt-nukebait.oramap a1111dda522c0e498dd39296e30460d013c70a4b http://resource.openra.net/maps/20139/
45. order-of-battle.oramap 129af89aa1b01751d051a0d6a73188cf2ae11d42 http://resource.openra.net/maps/20223/
46. ping-pong-bay.oramap c5b282f5d5bf15af88a1b0c55642b2cc0a833923 http://resource.openra.net/maps/13534/
47. pirates-and-emperors.oramap 95f22aafb9eaa85a7f01d95b3ecacb9c1f7040b3 http://resource.openra.net/maps/20392/
48. plan-b.oramap d431a14853306c14f6cf965d65b15b846b80fa28 http://resource.openra.net/maps/20394/
49. pressure-cnc.oramap 99db157af93a33b1148539aaecf3c0e54ca3810f http://resource.openra.net/maps/20395/
50. reasonable-doubt.oramap b6cefe97e71a9015876c395177516e2d237e1510 http://resource.openra.net/maps/12615/
51. riverside-frontline.oramap 6216f498cdf9e9e9db87415bb4bcf0972d1073e9 http://resource.openra.net/maps/20141/
52. scorched-earth.oramap c9bdfd8f71a96472aa7b06d124ad3cdcd9b1b6d8 http://resource.openra.net/maps/18504/
53. sustain.oramap 672b3668c26140d2f282768a2d4413680d21184f http://resource.openra.net/maps/20396/
54. tactical-advance.oramap e75c392ff23c327c25b61224b1524a649d893618 http://resource.openra.net/maps/17483/
55. thawed-front.oramap f561aaea8ce10996d118aa3e4854a461f2e628d3 http://resource.openra.net/maps/18035/
56. the-hot-box 7f6de317db131e2051d67ffe4ae300df3adf2656 http://resource.openra.net/maps/20150/
57. the-new-mandarins.oramap ce4fa5cfab658948d08b127f8de7ae3689fc11e8 http://resource.openra.net/maps/20397/
58. tiberian-falls.oramap ee94bc34598b56237c7a42ec2a6d3be2d5ea002a http://resource.openra.net/maps/20034/
59. tiberian-war.oramap 2cba43e4eabdfd9bc73ba3ede320d6b2723c7e93 http://resource.openra.net/maps/20398/
60. tiberium-canopy.oramap dab8fefee5db7e71c65083f77aa198f0673ab621 http://resource.openra.net/maps/18270/
61. tiberium-garden-2.oramap 9ba468cece35e4cfc5ede64cd3bac514869748a7 http://resource.openra.net/maps/20399/
62. too-hot.oramap a536dd088a5aa394e02628d1bf1f3573ae353e03 http://resource.openra.net/maps/11902/
63. trial-of-possession.oramap a8630d3836b63e05c8f981027f49006375725f59 http://resource.openra.net/maps/17198/
64. tropic-expanse.oramap 809910b462963f6265346ee0b08e17fb45fbb0e7 http://resource.openra.net/maps/20356/
65. twin-lakes.oramap d5d6262e6676b7a4168a929fd6bebc34226ea40e http://resource.openra.net/maps/17426/
66. twisted-peak.oramap 6bf1875739a8efa76d9bff57ae5ac41cb11c2351 http://resource.openra.net/maps/17184/
67. two-ponds.oramap 8d1334d692c199d74b933f26b0312ffb81ef2e31 http://resource.openra.net/maps/20400/
68. understanding-power.oramap a865ca55b6a7c61879fac87375589d3a5a9a91a6 http://resource.openra.net/maps/20401
69. valley-of-gold.oramap 45c741a4c83ed348ab4318e250249f6a8957401c http://resource.openra.net/maps/20153/
70. vectors-of-battle.oramap d771d3cc80b58cd6de4306aa42c145fe73da5537 http://resource.openra.net/maps/20154/
71. warchild.oramap a2531bd8b0a2cbbf2e2fe303797643d95e74a80e http://resource.openra.net/maps/20353/
72. warkraft.oramap 11cab3936e069f016896295ef86fcf18c00d0b3c http://resource.openra.net/maps/20402/
73. winters-end.oramap 3a1ce5ceea6f7dfc564ede5b86aaf3a5c45a274b http://resource.openra.net/maps/20157/
74. x896.oramap 084b4bddd86b79671048f3fd289e3c68c1ce4886 http://resource.openra.net/maps/13413/
75. year-501.oramap d7f75c0b952e88309c4dc1ba2c80ab5cff5df720 http://resource.openra.net/maps/20158/

Irnub commented:
Great work. Good to see that Echo of Damnation has been involved - he's really taken TD mapping to the next level.

ltem commented:
Impressive and so fast :) all my issues are solved so :+1:  from me.

obrakmann reviewed (approved):
👍

obrakmann commented:
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/d64dfc7181b71d1cb86aaa3826a20b37baf96602)
